packages/pagebuilder/lib/utils.js

```diff
@@ -12,7 +12,7 @@ export function getBackgroundImages(node) {
         backgroundSize: node.style.backgroundSize,
         backgroundPosition: node.style.backgroundPosition,
         backgroundAttachment: node.style.backgroundAttachment,
-        backgroundRepeat: node.style.backgroundRepeat !== 'no-repeat'
+        backgroundRepeat: node.style.backgroundRepeat
```

Comment:
Pointing out this one specifically used to make a determination of whether to repeat or not when there was no data available to make the decision. In other words, it used to default to `backgroundRepeat: true`, which - after these changes - would correspond to `backgroundRepeat: 'repeat'`.

Instead, the result of this change is `backgroundRepeat` gets set to an empty string.